src/components/Navigation/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as Navigation } from './Navigation';
```

Comment:
yep, but for the main `Navigation` element I got to do this because I export a variationable version and a normal one. Will change for others